### PR TITLE
Feature/controlled access

### PR DESCRIPTION
**Pull request by pinin4fjords**
Fed the correct parameters by the change in https://github.com/ebi-gene-expression-group/scxa-control-workflow/pull/16, requires use of of pre-downloaded files for controlled access analysis.

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -44,18 +50,21 @@ process download_fastqs {
     maxForks params.maxConcurrentDownloads
     time { 1.hour * task.attempt }
 
-    errorStrategy { task.attempt < 3 ? 'retry' : 'ignore' }   
+    errorStrategy { task.attempt < 3 && task.exitStatus != 2 ? 'retry' : 'ignore' }
```
Comment (suhaibMo) [inline main.nf:53]: Any particular reason why exit code 2 not 1 ?

**Review by suhaibMo — APPROVED** [review]
Overall LGTM though.

File: main.nf (hunk 1 above)
Comment (pinin4fjords) [reply main.nf:53]: See the process code below- the exit code of 2 is used to return an error indicating local files were not found in a controlled access experiment (and retries are therefore not worthwhile).

**Comment by pinin4fjords** [comment]
Thanks @suhaibMo for review

**Pull request merged**